gui-easy/gui/easy/scribblings/quickstart.scrbl

```diff
@@ -154,5 +164,11 @@ individual counter by passing in a derived observable to its
 
 @(define repo-link
   (link "https://github.com/Bogdanp/racket-gui-easy" "Git repository"))
+@(define easygui-publications-funarch23
```

Comment:
I don't remember the details offhand, but there are ways to put citations in Scribble documents (the Racket manuals do this, for example).

There may also be a better link than arXiv, although arXiv is potentially more accessible. See also links collected [in this Discourse post](https://racket.discourse.group/t/funarch-2023-functional-shell-and-reusable-components-for-easy-guis/2288).

Reply:
Yes, the accessibility was what I had in mind when I opted for Arxiv. 

When added, the bibliography section stood a bit bare. The `repo-link` is only used once across the docs so I thought I would turn that into a `bib-entry` too, which makes it more visible overall, especially when arriving to the docs via DrRacket's context sensitive help. There are not many citations in this document and links would appear more consistent and don't break the flow of reading. But in any case, we now have both versions and can decide which one to go for.